### DISCUSSION TITLE
Getting error while deploying (L2) zksync

### DISCUSSION THREAD

**Discussion by IshanJ9** [Cyfrin/foundry-full-course-cu#3022] [Q&A]
![Screenshot 2024-12-10 224228](https://github.com/user-attachments/assets/e9da5fbd-4ca4-4a3b-8d3a-94ea1a84c4db)

**Comment by EngrPips** [comment]
Hello @IshanJ9, Please put your contract file inside a `contracts` folder and also ensure you are working in the `default-workspace` of Remix. Also, clear your browser cache and retry. Let us know how things play out.
